week2

WanJu0 opened the pull request:
老師您好提交第二週作業
網址如下：http://54.64.33.230:3000/
再請老師幫忙看看是否還需要改進的地方
謝謝

WanJu0 committed:
week2

cwpeng reviewed (changes requested):
1. Layout exceeds screen width in small to middle size devices. There should not exist horizontal scroll bar in any situation.
2. Many details: search icon should be in center, font size, category block should be on top of other elements.
![54 64 33 230_3000_](https://user-images.githubusercontent.com/6104683/204491118-d6dffac8-ccec-4ddf-8082-7d433b9172b6.png)

WanJu0 committed:
week2 modify

WanJu0 committed:
week2 modify_2

WanJu0 committed:
week2 modify_3

WanJu0 committed:
week2 modify_4

WanJu0 committed:
week2 modify_5

WanJu0 committed:
week2 modify_6

WanJu0 committed:
week2 modify_7

WanJu0 commented:
hello teacher:
1.Modified the problem Layout exceeds screen width in small to middle size devices.
2.Modified the  search icon, font size and some minor errors.
please help to review it, thanks.

cwpeng reviewed (approved):
OK. Done. Keep Going.
